covasim/webapp/index.html

```diff
@@ -70,30 +70,30 @@ <h5 class="modal-title" id="exampleModalLabel">About Covasim</h5>
 
   <div class="content row">
 
-    <div id="parameters" class="col">
+    <div id="parameters" class="col-sm">
 
       <div class="row">
 
         <fieldset class="col-md-6">
```

Comment:
I wonder if we can move these fieldsets to a partial, there are 2 so far that are the same and I have a pr open that starts a 3rd. https://012.vuejs.org/api/elements.html#partial

Reply:
A component maybe better suited https://vuejs.org/v2/guide/#Composing-with-Components